code/code/misc/being.cc

```diff
@@ -1774,3 +1774,16 @@ bool TBeing::applyTattoo(wearSlotT slot, const sstring & tat, silentTypeT silent
   }
   return FALSE;
 }
+
+bool TBeing::canMeditate()
+{
+  if (isLinkdead() || (getPosition() < POSITION_RESTING))
+    return FALSE;
+  
+  if (getPosition() > POSITION_STANDING && 
+    !(getPosition() == POSITION_MOUNTED && 
+    doesKnowSkill(SKILL_ADVANCED_RIDING) && (getSkillValue(SKILL_ADVANCED_RIDING) >= 50))) {
```

Comment:
The doesKnowSkill looks redundant with getSkillValue. Is it possible to have skill value and still not know it?